docs/getting_started.md

```diff
@@ -3,42 +3,48 @@
 
 ## Prerequisites
 
-1. Python >= 3.6
-2. Configure AWS Account credentials when testing with AWS platforms,
-   check default section in `~/.aws/credentials` for access/secret key
-   [check aws-configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
-3. oc client binary is installed on your localhost and binary is listed in $PATH
-   (running oc version on terminal should display version > 3.11).
-   Latest client can be downloaded from [oc-client](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/).
-4. For vSphere based installations, [terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
-   and [jq]( https://stedolan.github.io/jq/download/) should be installed ( terraform version should be 0.11.13  )
-5. For UI testing follow the instructions in [openshift console](https://github.com/openshift/console#dependencies) repository.
-   After clone of repository and setting all dependencies please create configuration file with RUN section with `openshift_console_path`
-   parameter pointing to your local clone of openshift console repo. See example in `conf/ocsci/ui-testing.yaml`.
+* Python >= 3.7
```

Comment:
Why is python 3.6 not an option anymore?

Reply:
The rest of page refers to Python 3.7. The page should be consistent, I'm happy to refer to 3.6 consistently throughout the page, if we know it will work.

Reply:
I agree that we should be consistent. We currently run unit tests against 3.6 and AFAIK none of our code has a hard requirement on 3.7. I see the references to 3.7 are with regards to the virtual environment. We could change this to be more generic instead of stating that you should explicitly create a 3.7 venv. I'm also not opposed to just leaving your change as is and referring to python 3.7 as the minimum requirement. All of our automation is done using python 3.7 and I'm not personally aware of anyone who is still using 3.6.

Reply:
I only feel that the page should be consistent. As a newcomer to this repo I'm happy to take suggestions either way. I'll leave this conversation open to solicit input from others, but will continue making other improvements to this doc.

Reply:
I think for consistency we can leave `Python >= 3.6` in the Prerequisites and change the references to `python3.7` later in the doc to `python`.

Reply:
Sounds good to me.